source/chapters/microphones.rst

```diff
@@ -6,19 +6,22 @@ Using Microphones
 *****************
 There are three ways to use microphones with Mixxx, each with their own pros
 and cons:
-  * Software monitoring: cheapest and simplest to set up, but there is a
-    disorienting delay, referred to as "latency", between sound going into the
-    microphone and hearing it come out speakers or headphones
-  * Direct monitoring: avoids the latency of software monitoring but requires a
-    soundcard capable of direct monitoring and can be complicated to set
-    up. Recommended for most users.
-  * External mixing: most expensive option to set up with high sound quality
-
-.. warning:: USB microphones are not recommended because they cannot be used
+  * :ref:`Software monitoring<microphones-software-monitoring>`: cheapest and 
+    simplest to set up, but there is a disorienting delay, referred to as 
+    "latency", between sound going into the microphone and hearing it come out 
+    speakers or headphones
+  * :ref:`Direct monitoring<microphones-direct-monitoring>`: Recommended for 
+    most users. This avoids the latency of software monitoring but require a 
+    soundcard that supports this feature and can be complicated to set up.
+  * :ref:`Hardware mixers<microphones-hardware-mixers>`: most expensive 
+    option to set up with high sound quality
+
+.. warning:: USB microphones are not recommended because they can only be 
+             used with software monitoring with Mixxx. not be used
```

Comment:
Some spelling problems here.
I think this warning box is too big. Are those self monitoring mic's that common? I've never seen one!

Maybe rephrase like so:
Warning: USB mics are not recommended because they can't make use of the direct monitoring feature, thus introducing latency.

I think the bit about the mics with integrated sound card should be further analysed in DJ Hardware chapter, under microphones.

Reply:
These are really common. Lots of users come to the forum confused about how to configure them with Mixxx. I will think about how to make this warning box smaller and move the details to another place.